extensions/kafka-client/deployment/src/main/resources/dev-ui/qwc-kafka-add-topic.js

```diff
@@ -59,6 +59,14 @@ export class QwcKafkaAddTopic extends LitElement {
                         min="0" 
                         max="99">
                     </vaadin-integer-field>
+                    <vaadin-text-field
```

Comment:
I don't think that field would work to populate `configs` as `Map<String, String>`.

Reply:
@phillip-kruger ^